docs/ops.html

```diff
@@ -4054,16 +4054,19 @@ <h3>Reverting to ZooKeeper mode During the Migration</h3>
           </ul>
         </td>
         <td>
-          It is important to perform the <code>zookeeper-shell.sh</code> step quickly, to minimize the amount of
-          time that the cluster lacks a controller.
+          It is important to perform the <code>zookeeper-shell.sh</code> step <b>quickly</b>, to minimize the amount of
+          time that the cluster lacks a controller. Until the <code> /controller</code> znode is deleted,
```

Comment:
`Until the <code> /controller</code> znode is deleted, you can also ignore any errors in the broker log`

From my understanding, I think the error will disappear after broker roll into pure ZK mode, that is, the broker config doesn't contain voters info. Is that right?

Reply:
Yes. This is what I am saying in the next sentence `Those error logs should disappear after second roll to pure zookeeper mode.`